README.md

```diff
@@ -19,7 +19,10 @@ devtools::install_github("d3b-center/annoFuse", dependencies = TRUE)
 ### Prerequisites for cohort level analysis
 
  - merge calls from each caller for you cohort and a column `annots` with additional annotation (eg from FusionAnnotator or caller specific annotation we have used FusionAnnotator in our vignettes)
- - reference folder  with a gene [genelistreference.txt](https://github.com/d3b-center/annoFuse/blob/master/inst/extdata/genelistreference.txt) and [fusionreference.txt](https://github.com/d3b-center/annoFuse/blob/master/inst/extdata/fusionreference.txt) inst/extdata has reference files we've used in our vignettes
+
+ - reference folder  with a gene [genelistreference.txt](https://github.com/d3b-center/annoFuse/blob/master/inst/extdata/genelistreference.txt) and [fusionreference.txt](https://github.com/d3b-center/annoFuse/blob/master/inst/extdata/fusionreference.txt) inst/extdata has reference files we've used in our vignettes.
+The fusion reference is made up of TCGA fusions (Hu et al. 2018), gene reference file was created from publically available reference files : oncogene (http://www.bushmanlab.org/assets/doc/allOnco_Feb2017.tsv), tumor suppressor (Zhao et al. 2013)-(Zhao et al. 2016), kinase from pfam location database (http://hgdownload.soe.ucsc.edu/goldenPath/hg38/database/ucscGenePfam.txt.gz) and id description from pfamID Description database (http://hgdownload.soe.ucsc.edu/goldenPath/hg38/database/pfamDesc.txt.gz), as well as from Human Kinome database (Manning et al. 2002), transcription factor (Lambert et al. 2018), and  whether it has been reported in the Catalogue of Somatic Mutations in Cancer (COSMIC) Cancer Gene Census (Sondka et al. 2018). In addition,  we added the following genes specific to pediatric cancer from literature review; _MYBL1_ (Ramkissoon et al. 2013), _SNCAIP_ (Northcott et al. 2012), _FOXR2_ (Sturm et al. 2016), _TTYH1_ (Kleinman et al. 2014), and _TERT_ (Valentijn et al. 2015);(Lindner et al. 2015);(Karlsson et al. 2015);(Karsy et al. 2017)  were added to the oncogene list and _BCOR_ (Sturm et al. 2016) and _QKI_ (Bandopadhayay et al. 2016).
```

Comment:
```suggestion
The fusion reference is a compilation of:
  -  TCGA fusions (Hu et al. 2018)
  - gene reference file, which was created from publicly-available reference files
    - oncogene (http://www.bushmanlab.org/assets/doc/allOnco_Feb2017.tsv)
    - tumor suppressor (Zhao et al. 2013, Zhao et al. 2016)
    - kinase from pfam location database (http://hgdownload.soe.ucsc.edu/goldenPath/hg38/database/ucscGenePfam.txt.gz)
    - id description from pfamID Description database (http://hgdownload.soe.ucsc.edu/goldenPath/hg38/database/pfamDesc.txt.gz) and Human Kinome database (Manning et al. 2002)
    - transcription factor (Lambert et al. 2018)
    - Catalogue of Somatic Mutations in Cancer (COSMIC) Cancer Gene Census fusions (Sondka et al. 2018)
    - Oncogenes and tumor suppressor genes specific to pediatric cancer were added based on literature review; _MYBL1_ (Ramkissoon et al. 2013), _SNCAIP_ (Northcott et al. 2012), _FOXR2_ (Sturm et al. 2016), _TTYH1_ (Kleinman et al. 2014), and _TERT_ (Valentijn et al. 2015);(Lindner et al. 2015);(Karlsson et al. 2015);(Karsy et al. 2017) were added to the oncogene list. _BCOR_ (Sturm et al. 2016) and _QKI_ (Bandopadhayay et al. 2016) were added to the tumor suppressor list.

```

Reply:
@kgaonkar6 these citations are good, but the papers are not linked - can you just hyperlink to the pubmed page for all of them?